src/workouts.json

```diff
@@ -1,86 +1,115 @@
 {
-    "Classic Workout": [
+    "BJJ Mobility Warm Up Routine": [
```

Comment:
Currently, you're replacing the two existing workouts with the BJJ+Yoga routines - this change I can't accept in my `main` branch, I'd be happy to approve them if you'd just add them to the bottom if this list (so we'd have 4 workouts in total to choose from) 😎